Libs/Core/ctkAbstractJob.cpp

```diff
@@ -218,7 +219,7 @@ QString ctkAbstractJob::loggedText() const
 }
 
 //----------------------------------------------------------------------------
-void ctkAbstractJob::setLoggedText(QString loggedText)
+void ctkAbstractJob::addLoggedText(QString loggedText)
```

Comment:
`logText` or `addLog` would be simpler. Since we can only log text (and QString specifies that it is text anyway), `addLog` may be better.

```suggestion
void ctkAbstractJob::addLog(QString loggedText)
```